pkg/cluster/cluster_config.go

```diff
@@ -141,7 +141,7 @@ func GetRelease(ctx context.Context, cli client.Client) (Release, error) {
 		},
 	}
 	// Set platform
-	platform, err := GetPlatform(ctx, cli)
+	platform, err := getPlatform(ctx, cli)
```

Comment:
i am thinking
why not move  
```
	// get old release version before we create default DSCI CR
	oldReleaseVersion, _ := upgrade.GetDeployedRelease(ctx, setupClient)
```
up
then we do not even need to have this 
```
// Get operator platform
	platform, err := cluster.GetPlatform(ctx, setupClient)
	if err != nil {
		setupLog.Error(err, "error getting platform")
		os.Exit(1)
	}
```
but just 
`platform := oldReleaseVersion.Name`